doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -245,7 +245,7 @@ of the PG. These values can be important in a few cases:
 .. topic:: Placement Group IDs
 
    PG IDs consist of the pool number (not the pool name) followed 
-   by a period (.) and the PG ID-- (a hexadecimal number). You
+   by a period (.) and the PG ID--a hexadecimal number. You
```

Comment:
As written this is tail recursion, the PG ID consists of a number and a PG ID.

Suggest something like

>PG IDs consist of the pool number (not the pool name) followed by a period (.) and a hexadecimal number.

Reply:
Damn, Anthony, that is good. I spent the whole morning doing suggestion cleanup and clearly wasn't paying attention to the content of this sentence when I made this PR. Thank you for catching this.